source/public/javascripts/game.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+
+  $('#player1-container').on('change', function(event){
+    $('#player1-container').toggle(false);
+    $('#player2-container').toggle(true);
+  });
+
+  $('#player2-container').on('change', function(event){
+    // $('#MyGame').submit();
+
+    // $target = $(event.target)
```

Comment:
@gellieb Okay! Phew, so! the thing here is, your target is not the form. So it won't have the action and method attr! 

The event.target is always the dom element that fired the event. 

In this case it would be #player2-container